suite-native/formatters/src/components/AccountAddressFormatter.tsx

```diff
@@ -4,18 +4,34 @@ import { useSelector } from 'react-redux';
 import { AccountsRootState, selectAccountLabel } from '@suite-common/wallet-core';
 import { Text, TextProps } from '@suite-native/atoms';
 import { AccountKey } from '@suite-common/wallet-types';
+import { prepareNativeStyle, useNativeStyles, mergeNativeStyleObjects } from '@trezor/styles';
 
 import { FormatterProps } from '../types';
 
 type AccountAddressFormatterProps = FormatterProps<AccountKey> & TextProps;
 
-export const AccountAddressFormatter = ({ value, ...rest }: AccountAddressFormatterProps) => {
+const addressStyle = prepareNativeStyle(_ => ({
+    // ellipsizeMode="middle" is not working  on Android with negative letterSpacing defined in @trezor/theme typography.
+    letterSpacing: 0,
+}));
+
+export const AccountAddressFormatter = ({
+    value,
+    style,
+    ...rest
+}: AccountAddressFormatterProps) => {
+    const { applyStyle } = useNativeStyles();
     const accountLabel = useSelector((state: AccountsRootState) =>
         selectAccountLabel(state, value),
     );
 
+    const baseAddressStyle = applyStyle(addressStyle);
+    const mergedAddressStyle = style
+        ? mergeNativeStyleObjects([style, baseAddressStyle])
```

Comment:
You may try just fix types of `Text` to `style?: NativeStyleObject | NativeStyleObject[];` so you can pass array of styles, it should work without any other changes. If not don't bother with that and use this solution.

Reply:
Or maybe you can pass that array directly to `applyStyle([style, addressStyle])`

Reply:
None of these solutions fixed the problem. Changing `Text `props to accept a list of styles did not help because the `AccountAddressFormatter` is extending the `Text` props. So instead of these three lines merging the style objects, I would have to add a check if the incoming style is an array or a single style object. 

The second soluting did not work because the `style` is a `NativeStyleObject` ( `applyStyle` already applied) and is not compatible as a `applyStyle` parameter.

For this reason, i left the original solution since it is only three extra lines of code.